SPECK/src/main/java/SPEck.java

```diff
@@ -25,28 +25,43 @@
  */
 
 public class SPEck implements Serializable {
+
+    /**
+     * Initializer
+     * @param   scc    <JavaSparkContext> spark context for parallel execution
+     * @param   fileIn  <String> storing the file name of the input dataset to mine the SFSP
+     * @param   fileOut <String> storing the file name of the output (SFSP)
+     * @param   strategy <String> storing the method of sampling the random dataset
+     * @param   datasetFilePaths <String[]> storing the file paths of the dataset files
```

Comment:
Which dataset files? I think these are the *sampled* dataset files, but which ones? The `T` or the `P`? 
Please let me know and we'll figure out a better variable name and better comment.

Reply:
I think it is T so I named it `datasetTPaths` for now

Reply:
Can you call it `pveDatasetPaths`, `pve` will stands for `p-value estimation`.